### PR TITLE
[T2A2][T09-A3]Francis Cheng 

### PR DESCRIPTION
**Pull request by francischeng070**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -54,7 +54,7 @@
     /**
      * A platform independent line separator.
      */
-    private static final String LS = System.lineSeparator() + LINE_PREFIX;
+    private static final String lineSeparator = System.lineSeparator() + LINE_PREFIX;
```
Comment (okkhoy) [inline src/seedu/addressbook/AddressBook.java:57]: This is a good catch.

I encourage you to try find some more coding standard violations in the given code base.

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -1082,46 +1082,46 @@ private static boolean isPersonEmailValid(String email) {
...
+        return String.format(MESSAGE_COMMAND_HELP, COMMAND_CLEAR_WORD, COMMAND_CLEAR_DESC) + lineSeparator
+                + String.format(MESSAGE_COMMAND_HELP_EXAMPLE, COMMAND_CLEAR_EXAMPLE) + lineSeparator;
     }
 
     /** Returns the string for showing 'view' command usage instruction */
     private static String getUsageInfoForViewCommand() {
-        return String.format(MESSAGE_COMMAND_HELP, COMMAND_LIST_WORD, COMMAND_LIST_DESC) + LS
-                + String.format(MESSAGE_COMMAND_HELP_EXAMPLE, COMMAND_LIST_EXAMPLE) + LS;
+        return String.format(MESSAGE_COMMAND_HELP, COMMAND_LIST_WORD, COMMAND_LIST_DESC) + lineSeparator
+                + String.format(MESSAGE_COMMAND_HELP_EXAMPLE, COMMAND_LIST_EXAMPLE) + lineSeparator;
```
Comment (okkhoy) [inline src/seedu/addressbook/AddressBook.java:1124]: I notice that you haven't attempted the next activity, which is to abstract the specified method. Please try it if time permits :-)

**Review by okkhoy — COMMENTED** [review]
@francischeng070 some comments added. please check, ack and close the pull request. 
On an unrelated note, please add your photo on github to help us identify you :-)

File: src/seedu/addressbook/AddressBook.java (hunk 2 above)
Comment (francischeng070) [reply src/seedu/addressbook/AddressBook.java:1124]: Sorry, i have trouble with the refactor function in my eclipse. I will ask in my tutorial for assistance

**Pull request closed without merging**